ocrd_network/ocrd_network/rabbitmq_utils/publisher.py

```diff
@@ -30,9 +28,6 @@ def __init__(self, host: str = HOST, port: int = PORT, vhost: str = VHOST,
         if logger_name is None:
             logger_name = __name__
         logger = logging.getLogger(logger_name)
```

Comment:
The same as in the `consumer.py` case. This reminds me that the same change should happen in the `connector.py` as well when the `logger_name` parameter is removed.